### PR TITLE
Crypto-Market

### PR DESCRIPTION
**Pull request by Jaymelfah**
### In this project, I built a mobile web application to check a list of metrics (numeric values) from the CoinGecko API. This application displays a list of ranking cryptocurrencies and allows you to search on the home page for any of your favorite crypto coins. When users click one cryptocurrency they will be able to see things like CoinMarketCap rank, Current value, Symbol, price changes from 60d to 1h, and more.

- I used the CoinGecko API
- Designed the home page and the details page for the cryptocurrencies
- I added the filter feature on the home page 
- I displayed details of specific coins when the user clicks on them.
- I made the page responsive

- This is a link to the video presentation [https://www.loom.com/share/6ad034c4fa3a412ea29b11bd77a2ee6f](url)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,75 @@
...
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Website Preview
+![Preview](./src/images/Screenshot%202022-11-22%20at%204.19.31%20PM.png)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo
+Please use this [link](https://effervescent-stroopwafel-cbea62.netlify.app)
```
Comment (youmari) [inline README.md:28]: - [ ] your app works as expected locally however in the deployed version I can't any cards so please make sure that deployed version also working as expected 
- Yours 
Screenshot 2022-11-22 at 17 40 49

- Screenshot of the requirements 
Screenshot 2022-11-22 at 17 44 11

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Jaymelfah , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Jaymelfah) [reply README.md:28]: Hi @youmari . I have tested it with my cell phone and other devices and the link works. I suppose it is probably due to network issues when you first open it.

Comment (prolajumokeoni) [reply README.md:28]: Hi @Jaymelfah your deployed app works properly 👍🏿

**Comment by prolajumokeoni** [comment]
Hi @Jaymelfah 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**